src/containers/AddProfile/index.js

```diff
@@ -61,6 +65,15 @@ const AddProfile = ({
   const [confirmModal, setConfirmModal] = useState(false);
   const [isFormDirty, setIsFormDirty] = useState(false);
 
+  useEffect(() => {
+    form.setFieldsValue({
+      profileType: history?.location?.initialValues?.profileType,
+      name: history?.location?.initialValues?.name ? history?.location?.initialValues?.name : '',
+    });
+    setType(history?.location?.initialValues?.profileType);
+    setName(history?.location?.initialValues?.name ? history?.location?.initialValues?.name : '');
```

Comment:
why are you using history? just create a prop called initialValues and pass the values through it